app/lib/common/utilities/email_sender.dart

```diff
@@ -1,40 +1,47 @@
+import 'dart:io';
+import 'package:app/screens/history/history.dart';
+import 'package:flutter/cupertino.dart';
 import 'package:flutter_email_sender/flutter_email_sender.dart';
 import 'package:flutter/material.dart';
 
 class EmailSender {
-  Future<void> send(
-      String emailBody, String eSubject, String eRecipients) async {
+  String emailResponse;
+  Future<String> send(String emailBody, String eSubject, String eRecipients,
+      File attach) async {
     final Email email = Email(
       body: emailBody,
       subject: eSubject,
       recipients: [eRecipients],
-      attachmentPaths: null,
+      attachmentPaths: [
+        attach.path,
+      ],
       isHTML: false,
     );
-    String emailResponse;
-
     try {
       await FlutterEmailSender.send(email);
-      emailResponse = 'success';
+      emailResponse = 'Email Sent';
     } catch (error) {
       emailResponse = error.toString();
     }
-
-    print(emailResponse);
+    return emailResponse;
```

Comment:
I'm not sure you need to have the return type as `Future<String>` it can be just `String`. I think the `await` is blocking, so the function won't return until email response is set so it's not a true future.

The reason why what you have works is because dart is smart enough to wrap your string in a Future. This is useful for some things, but not futures.

- [ ] Change return type to just `String`